Calculate.Tests/ProgramTests.cs

```diff
@@ -0,0 +1,42 @@
+using Xunit;
+using System;
+using System.IO;
+
+namespace Calculate.Tests;
+
+public class ProgramTests
+{
+    [Fact]
+    public void Program_WriteLine_OutputIsEqual()
+    {
+        // Arrange
+        StringWriter capture = new();
+        Console.SetOut(capture);
+
+        Program testProg = new();
+
+        string testMessage = "foo";
+
+        // Act
+        testProg.WriteLine(testMessage);
+
+        // Assert
+        Assert.Equal(testMessage, capture.ToString().Trim());
+    }
+
+    [Fact]
+    public void Program_ReadLine_InputIsEqual()
+    {
+        // Arrange
+        string testMessage = "foo";
+        StringReader capture = new(testMessage);
+        Console.SetIn(capture);
+        Program testProg = new();
+
+        // Act
+        string? readString = testProg.ReadLine();
+
+        // Assert
+        Assert.Equal(testMessage, readString);
+    }
```

Comment:
I noticed your code coverage for Program is pretty low, the IntelliTect tool they briefly went over in class was pretty helpful for testing our main. https://github.com/IntelliTect/TestTools.Console
```suggestion
    }
    [Fact]
    public void Main_SampleUserInteraction_Success(){
        string view = @"Enter an equation in the format 'x + y' 
 or enter 'quit' to quit.<<2 + 2
>>
2 + 2  = 4";
        IntelliTect.TestTools.Console.ConsoleAssert.Expect(view, Program.Main);
    }
```
You might have to play with it a bit but here is an example for a successful calculation

Reply:
Will definitely look into this, thank you